k8s/helmfile/env/local/queryservice-ui.values.yaml.gotmpl

```diff
@@ -1,3 +1,3 @@
 image:
-  tag: "1.2"
```

Comment:
Not sure if there is a specific reason local was two versions behind, but when developing I ran 1.5 just fine, so I would assume it's an oversight.